### ISSUE TITLE
Issue ID: DAC_Programmatically_Hidden_Content1

### ISSUE THREAD

**Issue by VDuczekW** [guardian/dotcom-rendering#5062]
Report Link: https://drive.google.com/file/d/1pZnx1n-JqyqPu-58bqvODXczEaXoLzw9/view?usp=sharing

Full issue list: https://docs.google.com/spreadsheets/d/1vWQtZfD49uq7uNCNELuYN_HcIj-BKiE4t4Hwda3tJaQ/edit#gid=0

**Comment by bryophyta** [comment]
>**Note**
>This is an automatically-generated comment. Feel free to edit the content, but please don't edit this 'note' block.
 
# Intro to report

Copying the intro here to aid discoverability of the report's content:
```
Programmatically Hidden Content
Important information was programmatically hidden from screen reader users.
WCAG Reference:
1.3.1 Info and Relationships (Level A)
Understanding Info and Relationships | How to Meet Info and Relationships
Issue ID: DAC_Programmatically_Hidden_Content1
URL: https://www.theguardian.com/politics/audio/2018/mar/14/are-the-eus-draft-tradeguidelines-reasonable-brexit-means-podcast#maincontent
Page title: Are the EU's draft trade guidelines reasonable? Brexit Means… podcast | Politics
| The Guardian
Journey: 1.5
Screenshot:

When traversing the page using a screen reader it was identified that the information with
the date relating to the podcast did not get read out as expected. It was found that the date
had been hidden using the ‘aria-hidden="true"’ attribute. As this information is required for
users to understand the relevancy and accuracy of the podcast, it is essential that it is
available to all user groups.
```

**Comment by bryophyta** [comment]
It looks reasonably likely that this was introduced by mistake. `aria-hidden=true` used to be added only if the dateline component was 'secondary' (nb. I'm not sure what that meant in this context) and began to be added every time when the `secondary` property was being removed more generally.

I'm not aware of any reason why this content should be aria-hidden now.

That said, the dateline section does also include a _visually_ hidden 'date modified' field (see screenshot). If this is supposed to be visually hidden, then presumably it is also supposed to be aria-hidden. In which case we should remove aria-hidden from the parent, but apply it to the section that is currently visually hidden.


![Image](https://user-images.githubusercontent.com/37048459/199324009-bb719abc-8b3a-41a6-a8b9-fe4f8ba1ef81.png)
